server/index.js

```diff
@@ -369,10 +372,31 @@ app.get("/userLoggedInToMakeInst", (req, res) => {
 });
 
 app.get("/fbLoggedIn?", (req, res) => {
-  console.log(req.session.passport);
-  res.send(req.session.passport ? "true" : "false");
+  if (req.session.passport) {
```

Comment:
this block can be flattened --> promises r kewl.  Sequelize queries are promises, so return them and then move the nested '.then' outside the block